README.md

```diff
@@ -25,7 +25,7 @@ python -m venv venv
 source venv/bin/activate
 pip install -r requirements.txt
 pip install -r requirements-dev.txt
-pip install -e .
+pip install --editable .
```

Comment:
Don't see the need for this change, the others are fine

Reply:
+1, we are using -e in a lot of places

Reply:
Sure, I can remove the change if you insist. Please consider that people are accustomed to using short forms of options, but scripts are indeed more readable with longer, more descriptive options.

Reply:
This is a README file not a script, and hopefully folks working on a Python library would be familiar with this flag

Anyway, please update the commit history, it still shows a commit for the long option, thanks